### DISCUSSION TITLE
動態網頁爬蟲第一道鎖 — Selenium教學：如何使用Webdriver、send_keys(附Python 程式碼)

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#891] [優秀Python套件]
近兩堂課都是在學習網路爬蟲，好奇有甚麼同樣適合初學者學習的網路爬蟲套件，於是找到了適用動態網頁爬蟲的Selenium，雖然他是模擬消費者/使用者瀏覽頁面，因此速度相較起來較慢，但還是資處必學的網路爬蟲套件，這篇文有提供套件基本介紹，還有webdriver、send_keys這兩種方法的使用方式，可以輕鬆上手並實作練習，作者還有繼續介紹後續該套件各種方法的使用，有興趣都可以看該文章的內容進行學習，我覺得作者寫的很簡單明瞭，我不需要很多時間就可以學習一個新套件該如何使用。

https://medium.com/marketingdatascience/selenium%E6%95%99%E5%AD%B8-%E4%B8%80-%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8webdriver-send-keys-988816ce9bed

**Comment by Chiway1229** [comment]
哇 感謝你的分享
先把他存起來，有空時拿出來研究一波

**Comment by Kotori0501** [comment]
感謝分享
網路爬蟲初學者之一，剛接觸不久對很多相關語法都感到陌生，但有了這篇文章，或許能讓我更加了解某些網路文章上的程式碼再寫什麼了

**Comment by dabbie109303501** [comment]
感謝分享，這對python爬蟲的學習很有幫助，而且有這個語法整理很方便

**Comment by Yunhans** [comment]
今天好像就會教了，好期待

**Comment by ruthchen1020** [comment]
儲存起來，下次需要網路爬蟲的時候就抓出來看。

**Comment by quan0715** [comment]
其實有時候學習用 session 去模擬各種使用者行為也不錯，未必要透過Selenium，速度上還是會有些許落差
